src/jit/morph.cpp

```diff
@@ -6163,6 +6163,14 @@ GenTreePtr Compiler::fgMorphField(GenTreePtr tree, MorphAddrContext* mac)
             return newTree;
         }
     }
+    else if ((objRef != nullptr) && (objRef->OperGet() == GT_ADDR) && varTypeIsSIMD(objRef->gtGetOp1()))
```

Comment:
I'm still working on trying to figure out why we wind up in the `MACK_Ind` context. However, given that we do, I believe this is a conservative fix.